app/frontend/src/components/form/Step2.vue

```diff
@@ -604,6 +616,18 @@ export default {
       get() { return this.location.city; },
       set(value) { this.updateLocation({['city']: value}); }
     },
+    cityLatitude: {
+      get() {
+        return this.location.cityLatitude;
+      },
+      set(value) { this.updateLocation({['cityLatitude']: value}); }
+    },
+    cityLongitude: {
+      get() {
+        return this.location.cityLongitude;
+      },
```

Comment:
These gets don't need the contents in the curly braces to be expanded - can follow the style of the previous getters and setters.